src/backend/marsha/core/api.py

```diff
@@ -138,10 +139,16 @@ class TimedTextTrackViewSet(
 ):
     """Viewset for the API of the TimedTextTrack object."""
 
-    queryset = TimedTextTrack.objects.all()
     serializer_class = TimedTextTrackSerializer
     permission_classes = [IsRelatedVideoTokenOrAdminUser]
 
+    def get_queryset(self):
+        """Restrict list access to timed text tracks related to the video in the JWT token."""
```

Comment:
why not removing this route ? Or returning a 404 ? This is not really "rest" to retrieve a collection linked to a resource this way. The resource id must be in the URL. WDYT ?

Reply:
No in fact I don't know if what I wrote is good. `/api/timedtexttracks/{id}` should be a unique text track and not a collection.